system/doc/design_principles/sup-one_for_all.svg

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.0//EN" "http://www.w3.org/TR/2001/PR-SVG-20010719/DTD/svg10.dtd">
+<svg width="21cm" height="14.4cm" viewBox="-5 -5 330 220" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <style type="text/css">
+    .link {stroke: #000000; stroke-width: 2;}
+
+    .sup.label {font-size: 27px; fill: #000000; text-anchor: middle; font-family: monospace; font-style: normal; font-weight: bold;}
+    .sup.body {fill: #ddddff; fill-opacity: 1; stroke-width: 3; stroke: #000000;}
+
+    .child.label {font-size: 27px; fill: #000000; text-anchor: middle; font-family: monospace; font-style: normal; font-weight: bold;}
+    .child.label.index {font-size: 12px;}
+    .child.body {fill: #eeeeee; fill-opacity: 1; stroke-width: 3; stroke: #000000;}
+    .child.body.terminated {fill: #ff8888;}
+    .child.body.sup-terminated {fill: #ffaa00;}
+    .child.body.restarted {stroke: #00aa00;}
+
+    .ellipsis {fill: #000000; fill-opacity: 1; stroke: none;}
+    .ellipsis.sup-terminated {fill: #ffaa00;}
+    .ellipsis.restarted {stroke: #00aa00;}
+
+    .arrow {stroke-width: 2; stroke-linecap: round;}
+    .arrow.shutdown {stroke: #ffaa00;}
+    .arrow.restart {stroke: #00aa00;}
+
+    .divider {stroke: #000000; stroke-width: 0.2; stroke-dasharray: 1 3;}
+
+    .legend.label {font-size: 10px; fill: #000000; text-anchor: left; font-family: sans-serif; font-style: normal; font-weight: normal;}
```

Comment:
It would seem that `fop` does not like `text-anchor: left;`. When I remove it (and all the others in the other files), I still get the very similar svgs from what I can tell. Do you remember why you added it @Maria-12648430?

Reply:
Oh... that is supposed to be `start` (which is also the default), not `left` 🙄 TBH, I can't remember why I added it, probably a leftover from experimenting.

Since this PR is already merged, do you want me to open a new PR to fix it?

Reply:
I'll just merge a fix to master with it removed.

Reply:
https://github.com/erlang/otp/commit/f1b3f356b12c3b23b17f012b014f18e6728bfc72